ibis/backends/duckdb/__init__.py

```diff
@@ -732,6 +731,37 @@ def to_pyarrow(
         else:
             raise ValueError
 
+    @util.experimental
+    def to_torch(
+        self,
+        expr: ir.Expr,
+        *,
+        params: Mapping[ir.Scalar, Any] | None = None,
+        limit: int | str | None = None,
+        **kwargs: Any,
+    ) -> dict[str, torch.array]:
```

Comment:
```suggestion
    ) -> dict[str, torch.Tensor]:
```

even if we wish it were true...

Reply:
😂